packages/eds-core-react/src/components/Checkbox/Checkbox.docs.mdx

```diff
@@ -0,0 +1,72 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Checkbox } from '.'
+
+# Checkbox
+
+Allow users to select one or more items from a set, and can turn an option on or off.
+
+<Links 
+    ariaUrl="https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/"
+    figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=18442%3A38197"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/028476-selection-controls/b/86826a"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Checkbox/Checkbox.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+/>
+
+<Story id="inputs-selection-controls-checkbox--introduction" />
+<PropsTable />
+
+## Usage
+
+**Selection controls**
+
+<ul>
+    <li>Allow users to select options, make decisions and set preferences.</li>
+    <li>Are to be visible and understandable at a quick glance.</li>
+    <li>Are always accompanied with a clear label.</li>
+</ul>
+
+**Checkbox**
+
+<ul>
+    <li>It should be visible at a glance if a checkbox has been selected, and selected items should be more visually prominent than unselected items.</li>
```

Comment:
I feel like this is more a directive to us, the designers of the components and not so very useful to developers using it. They can not control this